generation/maisi/scripts/compute_fid_2-5d_ct.py

```diff
@@ -0,0 +1,655 @@
+    if model_name == "radimagenet_resnet50":
+        feature_network = torch.hub.load(
+            "Warvito/radimagenet-models", model="radimagenet_resnet50", verbose=True, trust_repo=True
+        )
+        suffix = "radimagenet_resnet50"
+    else:
+        import torchvision
+
+        feature_network = torchvision.models.squeezenet1_1(pretrained=True)
+        suffix = "squeezenet1_1"
+
+    feature_network.to(device)
+    feature_network.eval()
+
+    # -------------------------------------------------------------------------
+    # Parse shape/spacings
+    # -------------------------------------------------------------------------
+    t_shape = [int(x) for x in target_shape.split("x")]
+    target_shape_tuple = tuple(t_shape)
+    if enable_resampling:
+        rs_spacing = [float(x) for x in enable_resampling_spacing.split("x")]
+        rs_spacing_tuple = tuple(rs_spacing)
+        if local_rank == 0:
+            logger.info(f"resampling spacing: {rs_spacing_tuple}")
+    else:
+        rs_spacing_tuple = (1.0, 1.0, 1.0)
+
+    center_slices_ratio_final = enable_center_slices_ratio if enable_center_slices else 1.0
+    if local_rank == 0:
+        logger.info(f"center_slices_ratio: {center_slices_ratio_final}")
+
+    # -------------------------------------------------------------------------
+    # Prepare dataset 0
+    # -------------------------------------------------------------------------
+    output_root0 = os.path.join(output_root, data0_folder)
+    with open(data0_filelist, "r") as f0:
+        lines0 = [l.strip() for l in f0.readlines()]
+    lines0.sort()
+    lines0 = lines0[:num_images]
+
+    filenames0 = [{"image": os.path.join(data0_dataroot, f)} for f in lines0]
+    filenames0 = monai.data.partition_dataset(
+        data=filenames0, shuffle=False, num_partitions=world_size, even_divisible=False
+    )[local_rank]
+
+    # -------------------------------------------------------------------------
+    # Prepare dataset 1
+    # -------------------------------------------------------------------------
+    output_root1 = os.path.join(output_root, data1_folder)
+    with open(data1_filelist, "r") as f1:
+        lines1 = [l.strip() for l in f1.readlines()]
+    lines1.sort()
+    lines1 = lines1[:num_images]
+
+    filenames1 = [{"image": os.path.join(data1_dataroot, f)} for f in lines1]
+    filenames1 = monai.data.partition_dataset(
+        data=filenames1, shuffle=False, num_partitions=world_size, even_divisible=False
+    )[local_rank]
+
+    # -------------------------------------------------------------------------
+    # Build MONAI transforms
+    # -------------------------------------------------------------------------
+    transform_list = [
+        monai.transforms.LoadImaged(keys=["image"]),
+        monai.transforms.EnsureChannelFirstd(keys=["image"]),
+        monai.transforms.Orientationd(keys=["image"], axcodes="RAS"),
```

Comment:
What are these default transform values based on? Should we mention this somewhere? If users are working with a different dataset, they might need to adjust them accordingly.

Reply:
These transforms are inherited from auto3dseg. They are applicable to almost all 3D CT datasets.

Reply:
Got it.